Source/WebInspectorUI/UserInterface/Views/RecordingActionTreeElement.js

```diff
@@ -112,6 +116,28 @@ WI.RecordingActionTreeElement = class RecordingActionTreeElement extends WI.Gene
             return parameterElement;
         }
 
+        function copyTextForElement(element, swizzleType) {
+            const textContext = element.textContent;
```

Comment:
Style: We only use `const` for things that remain constant between Web Inspector sessions (i.e. actual constants).  Please use `let` instead.